client/electron/go_vpn_tunnel.ts

```diff
@@ -21,10 +21,8 @@ import {checkUDPConnectivity} from './go_helpers';
 import {ChildProcessHelper, ProcessTerminatedSignalError} from './process';
 import {RoutingDaemon} from './routing_service';
 import {VpnTunnel} from './vpn_tunnel';
-import {
-  TransportConfigJson,
-  TunnelStatus,
-} from '../src/www/app/outline_server_repository/vpn';
+import * as config from '../src/www/app/outline_server_repository/config';
```

Comment:
*nit* You can simply `import {TransportConfigJson} from ...` here, so you don't need to touch the code below.